### DISCUSSION TITLE
Doesn't respect JSON_DIAGNOSTICS?

### DISCUSSION THREAD

**Discussion by jsarasin** [pboettch/json-schema-validator#205] [Q&A]
Not sure if I have something misconfigured (I'm using VCPKG) but when i set JSON_DIAGNOSTICS to 1 the output from validator.set_root_schema(person_schema); when having a type error doesn't output a json pointer.
What use is the library if it provides no context of the problem?

**Comment by pboettch** [comment]
> Not sure if I have something misconfigured (I'm using VCPKG) but when i set JSON_DIAGNOSTICS to 1 the output from validator.set_root_schema(person_schema); when having a type error doesn't output a json pointer. What use is the library if it provides no context of the problem?

I'd need a hell of a lot of more info to help here?

Could you provide a schema which produces the error you expect to have?
